File: find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        num_with_even_digits = 0
        for num in nums:
            count = self.count_of_digits(num)
            if count % 2 == 0:
                num_with_even_digits += 1
        
        return num_with_even_digits
    
    def count_of_digits(self, num: int) -> int:
        count_digits = 0
        while num > 0:
            count_digits += 1
            num //= 10
        
        return count_digits

File: find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.go
func findNumbers(nums []int) int {
    even_no_digits := 0
    for _, num := range nums{
        count_digits := 0
        
        for num != 0{
            num /= 10
            count_digits++
        }
        
        if count_digits % 2 == 0{
            even_no_digits++
        }
    }
    
    return even_no_digits
}